cmd/dep/glide_importer.go

```diff
@@ -202,7 +202,8 @@ func (g *glideImporter) buildProjectConstraint(pkg glidePackage) (pc gps.Project
 	}
 	pc.Constraint, err = g.sm.InferConstraint(pkg.Reference, pc.Ident)
 	if err != nil {
-		return
+		g.logger.Printf("Unable to interpret revision specifier '%s' for package %s: %s, (using the 'any' constraint instead)\n", pkg.Name, pc.Ident, err.Error())
```

Comment:
* The wrong variable is being passed in as the first fmt argument. We want to print the value passed to `InferConstraint` which was `pkg.Reference`.
* Change "revision specifier" to "constraint" in the warning.